### DISCUSSION TITLE
Frage zu JSON Datenfeldern: is / id und dn

### DISCUSSION THREAD

**Discussion by nicolaslang1** [Digitaler-Impfnachweis/certification-apis#145] [Q&A]
Hallo ich hätte eine Frage zu folgenden Datenfeldern:

is : Hierbei wird ja der Aussteller des Zertifikats gemeint. Ist das der Arzt dessen SMCB steckt oder die Person die das PVS für die Erstellung benutzt.

id: Wie ergibt sich dieser Wert. Aktuell nutze ich IZ28215B da ich dies in einem Beispiel gesehen hatte. Was nutze ich hierfür in der PU.

sd: "dose_posint": {
      "description": "Dose Number / Total doses in Series: positive integer, range: [1,9]",
      "type": "integer",
      "minimum": 1,
      "maximum": 9

Laut Schmema kann dies von 1 - 9 gehen. Gilt dies dann auch für alle Impfstoffe z.B. Johnson und Johnson.

Vielen Dank schon im Voraus.

MfG

Nicolas

**Comment by nicolaslang1** [comment]
Ok ich konnte mir ein paar Fragen selbst beantworten.

is wird nicht vom PVS ausgefüllt, wird also leer beim API Aufruf übergeben.

id ist ein Zufallswert der laut dieser Doku generiert werden soll (https://ec.europa.eu/health/sites/default/files/ehealth/docs/digital-green-certificates_dt-specifications_en.pdf)

sd: soll je nach Produkt ausgewählt werden, also Astra 2, Johnson & Johnson 1 etc. Aber in der oben genannten Doku, steht noch die Anmerkung, dass wenn jemand bereits Covid 19 hatte, dass sich die maximale Dosis anpassen soll. Also dann die maximale Dosis bei Astra z.B. nur noch 1 ist, er somit als voll geimpft gilt. 

Soll das schon so implementiert werden oder kommt das erst später? @oliver-steinbrecher 

Das würde ja heißen, dass man erst ein Genesenenzertifikat über das PVS erstellen müsste, oder im Formular eine Option "ist genesen" angeben muss und danach dass Impfzertifikat erstellt, damit das PVS bescheid weiß, dass die maximale Dosenanzahl verringert werden muss, da sonst dies Person nichts nützt, da sie keinen vollen Impfschutz hätte laut QR-Code.

**Reply by thinkberg** [reply]
Wichtig ist: Auch wenn die Input Daten fuer die API an das DCC Schema angelehnt sind, sind diese nicht identisch. D.h. Felder wie "is" und "co" sind beim API Aufruf nicht vorgesehen, da diese vom Backend automatisiert in das finale DCC eingetragen werden. "id" ist bei PVS Systemen die BSNR oder eine anderer eindeutige Nummer, die in der UVCI mit als ausgebende Stelle uebernommen wird (nicht verwechseln mit "is").

Die definition, ob ein Impfstoff mit 1/1, 2/2, etc. fuer Kreuzimpfungen oder Genesene eingetragen wird ist etwas, was durch das RKI definiert wird.

**Reply by ghost** [reply]
Siehe jedoch #148
